pandas/tests/series/test_arithmetic.py

```diff
@@ -43,6 +43,40 @@ def test_ser_flex_cmp_return_dtypes_empty(self, opname):
         result = getattr(empty, opname)(const).get_dtype_counts()
         tm.assert_series_equal(result, Series([1], ['bool']))
 
+    @pytest.mark.parametrize('op', [operator.eq, operator.ne,
+                                    operator.le, operator.lt,
+                                    operator.ge, operator.gt])
+    @pytest.mark.parametrize('names', [(None, None, None),
+                                       ('foo', 'bar', None),
+                                       ('baz', 'baz', 'baz')])
+    def test_ser_cmp_result_names(self, names, op):
+        # datetime64 dtype
```

Comment:
did any of these raise / not work before?

Reply:
None of these work at the moment.  They currently retain the series's name.